pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -47,6 +47,10 @@ func ValidateKubeSchedulerConfiguration(cc *config.KubeSchedulerConfiguration) f
 		allErrs = append(allErrs, field.Invalid(field.NewPath("percentageOfNodesToScore"),
 			cc.PercentageOfNodesToScore, "not in valid range 0-100"))
 	}
+	if cc.PodMaxBackoffDurationSeconds < 0 {
```

Comment:
Should be `<= 0` according to our code:

https://github.com/kubernetes/kubernetes/blob/4b19c3993bd801d9a415432372ff0af5b46624c3/pkg/scheduler/internal/queue/scheduling_queue.go#L216